src/maze_solver.py

```diff
@@ -49,18 +50,29 @@ def solveAStar(
     print(path) # prints [(0, 0, 0), (0, 1, 0), (0, 2, 0)]
```

Comment:
This docstring does not reflect the actual usage now, i recommend removing the example and changing the documentation.

Reply:
done